database/methods.js

```diff
@@ -4,20 +4,38 @@ const driver = neo4j.driver("bolt://localhost:7687", neo4j.auth.basic("neo4j", "
 
 const getSong = (song_id, res) => {
   var session = driver.session();
-  const cypher = 'MATCH (song:Song {id: {id} }) RETURN song';
   const params = {id: song_id};
-  console.log(params);
-  return session
-    .run(cypher, params)
+  
+  return Promise.all([
```

Comment:
just curious - is this a native javascript feature, you didn't need to import bluebird?

Reply:
From what I've seen, yes. There's no necessary requirements.